Extension/src/LanguageServer/extension.ts

```diff
@@ -228,6 +228,12 @@ export async function activate(): Promise<void> {
     ];
     codeActionProvider = vscode.languages.registerCodeActionsProvider(selector, {
         provideCodeActions: async (document: vscode.TextDocument, range: vscode.Range, context: vscode.CodeActionContext): Promise<vscode.CodeAction[]> => {
+            const ports: string[] = await lookupIncludeInVcpkg(document, range.start.line);
```

Comment:
I don't think this should be the first early return check.  I would make this the last one (right before `telemetry.logLanguageServerEvent`)